extensions-core/druid-basic-security/src/main/java/io/druid/security/basic/BasicAuthClassCompositionConfig.java

```diff
@@ -22,6 +22,14 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
+/**
+ * Basic authentication storage/cache/resource hadler config.
+ * The config is an option to specify classes of user/role managers, caches and notifiers.
```

Comment:
I would put it "BasicAuthClassCompositionConfig provides options to specify authenticator classes..."